### PR TITLE
Updates from next

### PR DESCRIPTION
**Pull request by Jephuff**

### CODE REVIEW

**Comment by Jephuff** [comment]
This is ready for review. Oncogrid isn't working with d3v4 but I will make a separate PR for that.

File: modules/node_modules/@ncigdc/components/SurvivalPlotWrapper.js [hunk 1]
```diff
@@ -31,26 +32,31 @@ const styles = {
...
 
-function onMouseEnterDonor(e, donor) {
-  const tooltip = document.querySelector('.global-tooltip');
-  tooltip.classList.add('active');
-  tooltip.innerHTML = `
-    ${donor.id}<br>
-    Survival Rate: ${Math.round(donor.survivalEstimate * 100)}%<br>
-    ${donor.censored ? `Interval of last follow-up: ${donor.time}` : `Time of Death: ${donor.time}`}
-  `;
+function onMouseEnterDonor(e, donor, dispatch) {
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/components/SurvivalPlotWrapper.js]: How is this(and onMouseLeaveDonor) getting dispatch, I don't see anything hooking it up?

Comment (alex-wilmer) [inline modules/node_modules/@ncigdc/components/SurvivalPlotWrapper.js]: my mistake, should be good now

Comment (alex-wilmer) [inline modules/node_modules/@ncigdc/components/SurvivalPlotWrapper.js]: well.. whenever github catches up and sees the new commit

**Pull request merged**